migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc

```diff
@@ -11,9 +11,15 @@ You can migrate application workloads by adding your clusters and replication re
 If your cluster or replication repository are secured with self-signed certificates, you can create a CA certificate bundle file or disable SSL verification.
 
 include::modules/migration-creating-ca-bundle.adoc[leveloffset=+1]
+
+[id='configuring-migration-plan_{context}']
+== Configuring a migration plan
+
```

Comment:
I'd add a sentence or two describing these modules to introduce the subsection. Back-to-back headings also render oddly.